test/docfx.Test/lib/GlobUtilityTest.cs

```diff
@@ -69,10 +72,6 @@ public void MatchFilesUsingGlobPattern(string pattern, string files, bool match)
         [InlineData("a[\\\\b]c")]
         [InlineData("{{a,b}}")]
         [InlineData("z{a,b{,c}d")]
-        [InlineData("**/")]
-        [InlineData("\\[*")]
-        [InlineData("a{b,c,d}e{d}{}")]
```

Comment:
we are not going to support these?

Reply:
these are supported

Reply:
would we add these supported test cases to above test?

---
In reply to: [229163945](https://github.com/dotnet/docfx/pull/3541#discussion_r229163945) [](ancestors = 229163945)

Reply:
liken this one: a{b,c,d}e{d}{}

---
In reply to: [229583124](https://github.com/dotnet/docfx/pull/3541#discussion_r229583124) [](ancestors = 229583124,229163945)

Reply:
sure